sdcm/monitorstack/__init__.py

```diff
@@ -359,6 +359,8 @@ def start_dockers(monitoring_dockers_dir, monitoring_stack_data_dir, scylla_vers
     cmd = dedent("""cd {monitoring_dockers_dir};
             echo "" > UA.sh
             ./start-all.sh \
+            --no-renderer \
```

Comment:
I think I would remove this, and test again.
cause it can break old monitoring versions (i.e. downloading old version which didn't had those flags)

Reply:
added code to identify if flag are available or not.